vnext/ReactWindowsCore/DevSettings.h

```diff
@@ -45,10 +46,16 @@ enum class JSIEngineOverride : int32_t {
   Last = V8Lite
 };
 
+struct ScriptMetadata {
+  uint64_t scriptVersion;
+  std::string bytecodeFilename;
+};
```

Comment:
This is chakra only setting, should it be called ChakraScriptMetadata? Also, we are using "bundle" and "script" interchangeably. It would be nice to standardize, eg "jsBundle", like we do in ReactNativeHost code.

Reply:
I think both of these make sense. I can kind of get changing the name to script in the lower levels of abstraction, but this is not at that low level.

---
In reply to: [342726192](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342726192) [](ancestors = 342726192)